include/xrpl/protocol/MPTIssue.h

```diff
@@ -66,6 +68,25 @@ operator!=(MPTIssue const& lhs, MPTIssue const& rhs)
     return !(lhs.mpt_ == rhs.mpt_);
 }
 
+MPT
+getMPT(uint192 const&);
+
+inline MPT
+badMPT()
```

Comment:
It would be better if we didn't need a sentinel value for a bad mpt. Is this really needed?